src/cljs/witan/ui/components/shared.cljs

```diff
@@ -123,3 +177,65 @@
   {:inspect-data true
    :frame true
    :history false})
+
+(defcard inline-group
+  (sab/html
+   [:ul
+    [:li (inline-group {:kixi.group/name "Foo Bar" :kixi.group/type :user})]
+    [:li (inline-group {:kixi.group/name "Foo Bar" :kixi.group/type :group})]]))
+
+(def states
+  [{:name "Alabama"}
+   {:name "Alaska"}
+   {:name "Arizona"}
+   {:name "Arkansas"}
+   {:name "California"}
+   {:name "Colorado"}
+   {:name "Connecticut"}
+   {:name "Delaware"}
+   {:name "Florida"}
+   {:name "Georgia"}
+   {:name "Hawaii"}
+   {:name "Idaho"}
+   {:name "Illinois"}
+   {:name "Indiana"}
+   {:name "Iowa"}
+   {:name "Kansas"}
+   {:name "Kentucky"}
+   {:name "Louisiana"}
+   {:name "Maine"}
+   {:name "Maryland"}
+   {:name "Massachusetts"}
+   {:name "Michigan"}
+   {:name "Minnesota"}
+   {:name "Mississippi"}
+   {:name "Missouri"}
+   {:name "Montana"}
+   {:name "Nebraska"}
+   {:name "Nevada"}
+   {:name "New Hampshire"}
+   {:name "New Jersey"}
+   {:name "New Mexico"}
+   {:name "New York"}
+   {:name "North Carolina"}
+   {:name "North Dakota"}
+   {:name "Ohio"}
+   {:name "Oklahoma"}
+   {:name "Oregon"}
+   {:name "Pennsylvania"}
+   {:name "Rhode Island"}
+   {:name "South Carolina"}
+   {:name "South Dakota"}
+   {:name "Tennessee"}
+   {:name "Texas"}
+   {:name "Utah"}
+   {:name "Vermont"}
+   {:name "Virginia"}
+   {:name "Washington"}
+   {:name "West Virginia"}
+   {:name "Wisconsin"}
+   {:name "Wyoming"}])
```

Comment:
:) This is just for the `defcard` which shows off the indexing. I really need some way to chop this out and only include in the devcard build.....